### ISSUE TITLE
Moving GTK apps mostly off-screen on the right or bottom screen edges makes the window disappear

### ISSUE THREAD

**Issue by git-f0x** [pop-os/cosmic-comp#373]
When a GTK app window (including Firefox) is moved past the right or bottom screen edge so that most of it is off-screen, the app window disappears past a certain point and reappears when going back.
This issue was present for a while, but can now interfere with the tiling regions.
Recording of the issue:

https://github.com/pop-os/cosmic-comp/assets/150025636/7a3a103b-758e-4c13-8d9f-6026898856d8

**Comment by ryanabx** [comment]
Are you on the latest commit 929f4fcb054710ee73d781f7050f0a005981fa0e ?

**Comment by git-f0x** [comment]
Yes. It was present for a long time, not related to any recent commits.

**Comment by ryanabx** [comment]
Wanted to make sure the newest commit didn't fix it

**Comment by Kartonrealista** [comment]
Now GTK apps seem to also disappear when dragged in tiling mode.

**Comment by Kartonrealista** [comment]
https://github.com/user-attachments/assets/69af5b91-edc6-4847-a7bc-49b262b90f7a

**Comment by git-f0x** [comment]
>Now GTK apps seem to also disappear when dragged in tiling mode.

I can't reproduce this (at least with Firefox and Nautilus), so I'm not sure if it's dependent on some settings or hardware.

**Comment by Kartonrealista** [comment]
> >Now GTK apps seem to also disappear when dragged in tiling mode.
> 
> I can't reproduce this (at least with Firefox and Nautilus), so I'm not sure if it's dependent on some settings or hardware.

I can't really test it right now since I don't have access to my PC until Monday, this may have been resolved while I was away. I'm using AMD CPU+GPU combo, so I'm not sure it's a driver issue.

**Comment by Kartonrealista** [comment]
The issue I reported seems to be unrelated to this, but to export of themes to GTK apps, when the given COSMIC theme contains has background alpha < 1. Since you can't edit transparency in COSMIC Settings -> Appearance right now and instead have to edit the themes .ron file directly I can't expect it to work right now.

**Comment by LiamillionSS** [comment]
Cosmic 1.0.4 still exhibits this with GTK apps, but it requires the app to be outside of snapping distance which is a lot further off screen compared to when it was initially reported.

**Comment by KodeBarista** [comment]
The only direction in which I could reproduce this was toward the bottom edge of the screen. However, as soon as I let go of the window, it snapped to the last tile recommendation that was shown before the window disappeared.
